### ISSUE TITLE
Feature Requets: Passive mode

### ISSUE THREAD

**Issue by bcoles** [SimplySecurity/SimplyEmail#51]
It would be nice to have a `--passive` mode, that disables all modules which interact with the target domain, and enables all modules which do not interact with the target domain.

Currently, to do this requires specifying individual modules with `-t`.

**Comment by whoot** [comment]
Are there even modules which interact with the target domain?
I mean all results are coming from search engines, arent they?

**Comment by bcoles** [comment]
@whoot

Yes, there are modules which interact with the target domain, for example:

* The [HtmlScrape module](https://github.com/SimplySecurity/SimplyEmail/blob/master/Modules/HtmlScrape.py) "scrapes the target website for emails and data".
* The Google modules search Google for documents which may contain email addresses, then download these documents. The module specifically avoids downloading modules from Google's cache. As such, the module may download documents from the target domain.
* The Exalead modules likely also operate in a similar fashion to the Google modules, although I haven't checked.

While it's possible to specify individual modules with `-t`, or modify the INI configuration file to disable some modules, this requires knowledge of which modules interact with the target domain.

It's not immediately obvious to the operator which modules interact with the target domain and which modules do not.

As such, it would be nice to have a `--passive` mode, that runs all modules which do not interact with the target domain.

**Comment by whoot** [comment]
Alright, thanks for clarification.
This stuff is indeed not easy to spot. Passive mode would be great!
